docs/ConfigurationFile.md

```diff
@@ -5,6 +5,8 @@ Deepforest uses a config.yml to control hyperparameters related to model trainin
 Deepforest comes with a sample config file, deepforest_config.yml. Edit this file to change settings while developing models. Please note that if you would like for deepforest to save the config file on reload (using deepforest.save_model), 
```

Comment:
@Om-Doiphode thank you for this Pr, let's add those details in line 5.  Something like:

DeepForest includes a default sample config file named deepforest_config.yml. Users have the option to override this file by creating their own custom config file. Initially, DeepForest scans the current working directory for the file. If it's not found there, it automatically resorts to using the default configuration.

You can edit this file to adjust settings while developing models. Please note that if you wish for DeepForest to save the config file upon reload (using deepforest.save_model),